templates/base.html

```diff
@@ -42,6 +42,7 @@
         <script type="text/javascript" src="{{ media_url }}html_player/player.js"></script>
         {% google_analytics %}
         {% endblock %}
+        {% block extra-head %}{% endblock %}
```

Comment:
Don't do this... use {{block.super}}: https://docs.djangoproject.com/en/1.7/topics/templates/